pkg/infrastructure/services/deployer_test.go

```diff
@@ -53,7 +53,7 @@ func Test_serviceDeployer_Deploy(t *testing.T) {
 	deployer := NewSingletonServiceDeployer(definition, serviceList, cli, meta.GetRegisteredSchema())
 	reconcileAfter, err := deployer.Deploy()
 	assert.NoError(t, err)
-	assert.True(t, reconcileAfter > 0) // we just deployed Infinispan
+	assert.Equal(t, time.Duration(0), reconcileAfter) // we just deployed Infinispan
```

Comment:
maybe rephrase this comment? didn't see infinispan getting deployed.

Reply:
done